harness/determined/core/_log_shipper.py

```diff
@@ -21,7 +21,7 @@ def __init__(
         session: api.Session,
         trial_id: int,
         task_id: str,
-        distributed: Optional[core.DistributedContext] = None
+        distributed: Optional[core.DistributedContext] = None,
```

Comment:
👏

Reply:
Courtesy of `pre-commit` (and whichever Python tool we use for that)!